memor/session.py

```diff
@@ -84,6 +84,33 @@ def __iter__(self):
         """
         yield from self._messages
 
+    def __add__(self, other_object):  # TODO: This method should raise a TypeError for other_object != (Response, Pormpt, Session)
+        """
+        Addition method.
+
+        :param other_object: other object
+        :type other_object: any
+        :return: new Session
+        """
+        if isinstance(other_object, (Response, Prompt)):
+            new_messages = self._messages + [other_object]
+            return Session(title=self.title, messages=new_messages)
+        if isinstance(other_object, Session):
+            new_messages = self._messages + other_object._messages
+            return Session(messages=new_messages)
+
+    def __radd__(self, other_object):  # TODO: This method should raise a TypeError for other_object != (Response, Pormpt)
+        """
+        Reverse addition method.
+
+        :param other_object: other object
+        :type other_object: any
+        :return: new Session
+        """
+        if isinstance(other_object, (Response, Prompt)):
+            new_messages = [other_object] + self._messages
+            return Session(title=self.title, messages=new_messages)
```

Comment:
So then what happens if we have and right-addition of a Session to another Session? Does it count as the left addition (add) for the first session?